internal/repository/location.go

```diff
@@ -0,0 +1,22 @@
+package repository
+
+import (
+	"go-rest-template/internal/domain"
+	"log"
+)
+
+type LocationRepository interface {
+	Create(location domain.Payload)
+}
+
+type locationRepository struct {
+}
+
+func NewLocationRepository() LocationRepository {
+	return &locationRepository{}
+}
+
+func (r *locationRepository) Create(location domain.Payload) {
```

Comment:
Acá podés recibir el puntero, asi no se duplica en memoria.
```suggestion
func (r *locationRepository) Create(p *domain.Payload) {
```

Reply:
Hecho [9146f46](https://github.com/luxarts/hte-location-ms/pull/1/commits/9146f4649d810a3b7223f506eea9124573628496)